modules/apps/29-fee/types/events.go

```diff
@@ -2,9 +2,15 @@ package types
 
 // 29-fee events
 const (
-	EventTypeIncentivizedPacket = "incentivized_ibc_packet"
+	EventTypeIncentivizedPacket        = "incentivized_ibc_packet"
+	EventTypeRegisterPayee             = "register_payee"
```

Comment:
just thinking aloud would there be a benefit to prefixing these events with say `ics29_register_payee`. In the future, there may be similar events from other modules it might become harder to distinguish?

Reply:
When we emit events we should always be including the `sdk.AttributeKeyModule` with the module name as well so it can be correlated like that pretty easily too I guess!